image.go

```diff
@@ -1145,7 +1156,7 @@ func (i *Image) SubImage(r image.Rectangle) image.Image {
 		i.subImageCache = map[image.Rectangle]*Image{}
 	}
 	i.subImageCache[r] = img
-	img.atime.Store(Tick())
+	img.updateAccessTime()
```

Comment:
Why is this change needed?

Reply:
I just wanted to know this change was just a refactoring and unrelated to the core focus.

> Also it is more expressive (speaking? Is that the phrase in english?). 

You mean expensive?

Reply:
You don't have to revert this. Thanks,